gulpfile.js

```diff
@@ -1,5 +1,5 @@
 /* jshint node: true */
-'use strict';
+'use strict'; // jshint ignore:line
```

Comment:
No idea what is going on with my setup but this is how the issue happens when the line is omitted.

https://gist.github.com/nepeat/0a3f934ba7beb040bca5

# Versions
* Gulp - 3.9.0
* gulp-jshint - 1.11.2
* jshint - v2.7.0